content/docs/studio/user-guide/billing-and-payment.md

```diff
@@ -1,70 +1,65 @@
-team level. You can
-[change the plan](/doc/studio/change-team-plan-or-size#change-team-plan) in a
-team's settings page. Each team is on its own plan and needs to be managed
-separately.
+When you sign up, you're on the **Free plan** by default. To switch to the
+**Team plan**, [create a team] first, then go to the [team's settings] and click
+on `Change plan` in the `Plan and billing` section. switch to the **Enterprise
+plan**, please [contact us].
 
 ## How total price is calculated
 
 For Team plans, invoice amount is calculated based on the number of seats in the
 team. All invoice amounts are calculated for one billing period at a time. A
-billing period is one month long. The price per seat can be found in the
-Iterative Studio [pricing page](https://studio.iterative.ai/pricing).
-
-Below is an example calculation:
+billing period is one month long (30 days). The price per seat can be found in
+the Iterative Studio [pricing page](https://studio.iterative.ai/pricing).
 
-Suppose you have a team with 5 seats. At a monthly rate of $40 per seat, the
-total payable amount for the team = $40 \* 5 = $200 per month.
+For example, suppose you have a team with 5 seats. At a monthly rate of $40 per
+seat, the total payable amount for the team would be $40 x 5 = $200 per month.
 
 Increasing or decreasing the number of seats during a billing period will result
-in pro-rated amount calculation. See the
-[next section](#how-increasing-or-decreasing-the-number-of-seats-affects-total-cost)
-for an example.
+in pro-rated amount calculation (see the next section for an example).
 
 For Enterprise plans, all invoice amounts are calculated based on the agreement
 between you and Iterative.
 
-## How increasing or decreasing the number of seats affects total cost
+## How increasing or decreasing the number of seats affects total price
 
 If you add members to available seats or remove members without reducing the
-number of seats in your team, there will be no impact in the cost to you.
+number of seats in your team, there will be no price impact.
 
 If you increase the number of seats, your account will be charged a prorated
 amount based on the time remaining in the billing period at the time each seat
 was added. For instance, if your billing period runs from the 15th of last month
 to the 14th of this month, then, if you add 2 seats on the 10th of this month,
-you will be charged for only 5 days for the 2 new seats. In the following
-billing period, the 2 additional seats will be taken into account for the entire
-billing period.
+you will be charged for only 5 days of the 2 new seats. In the following billing
+period, the 2 additional seats will be charged for the entire billing period.
 
-If you reduce the number of seats, you will accordingly receive an account
-credit that you can apply on your next payment.
+If you reduce the number of seats, you will receive an account credit for the
+prorated difference. You can apply it on your next payment.
```

Comment:
Hi @tapadipti 👋🏼 

How do users apply credits to next payments? Is there a special button? Is it automatic? Thanks

Reply:
It is automatic.

Reply:
Got it. Updated text to reflect so in c8e1480b5.